docs/user/alerting/action-types/jira.asciidoc

```diff
@@ -33,37 +33,21 @@ API token (or password)::  Jira API authentication token (or password) for HTTP
      apiToken: tokenkeystorevalue
 --
 
-[[jira-connector-config-properties]]
-**`config`** defines the action type specific to the configuration and contains the following properties:
+Config defines information for the action type.
 
-[cols="2*<"]
-|===
+`apiUrl`:: An address that corresponds to *URL*.
+`projectKey`:: A key that corresponds to *Project Key*.
 
-| `apiUrl`
-| An address that corresponds to *URL*.
+Secrets defines sensitive information for the action type.
 
-| `projectKey`
-| A key that corresponds to *Project Key*.
-
-|===
-
-**`secrets`** defines sensitive information for the action type and contains the following properties:
-
-[cols="2*<"]
-|===
-
-| `email`
-| A string that corresponds to *Email*.
-
-| `apiToken`
-| A string that corresponds to *API Token*. Should be stored in the <<creating-keystore, {kib} keystore>>.
-
-|===
+`email`:: A string that corresponds to *Email*.
+`apiToken`:: A string that corresponds to *API Token*. Should be stored in the <<creating-keystore, {kib} keystore>>.
 
+[float]
 [[jira-action-configuration]]
 ==== Action configuration
 
-Jira actions have the following configuration properties:
+Jira actions have the following configuration properties.
 
 Subaction::        The subaction to perform: `pushToService`, `getIncident`, `issueTypes`, `fieldsByIssueType`, `issues`, `issue`, or `getFields`.
 Subaction params:: The parameters of the subaction.
```

Comment:
Should the subactions be in alphabetical order?

Putting the subactions and params in a table might make this section easier to read.  I have the code if you want to use it.

<img width="840" alt="Screen Shot 2021-02-27 at 10 51 20 AM" src="https://user-images.githubusercontent.com/33642766/109397413-47ab7a00-78eb-11eb-951d-ff77fce23a38.png">

Reply:
issues, issue, and issueTypes are all very similar.  Are those really the names of different subactions?

Reply:
"Configuring and testing Jira" should be "Configure Jira" as per the template.

Reply:
Realized that the subaction params should be not user facing and have reverted the changes to add them to the user docs in this PR: https://github.com/elastic/kibana/pull/92878